examples/secret.py

```diff
@@ -0,0 +1,98 @@
+import typing
+
+import discord
+from discord.ext import commands
+
+bot = commands.Bot(command_prefix=commands.when_mentioned, description="Nothing to see here!")
+
+@bot.event
+async def on_command(ctx):
+    try:
+        await ctx.message.delete(delay=5)
+    except discord.HTTPException:
+        # Oh well, looks like we don't have the ability
+        # to hide the secret.
+        pass
+
+# the `hidden` keyword argument hides it from the help command. 
+@bot.group(invoke_without_command=True, hidden=True)
```

Comment:
Add comments about using groups perhaps. I don't think groups and subcommands have been used in many other examples.